script.js

```diff
@@ -1,4 +1,405 @@
+/*
+Introduction
+============
+1. Implement a simplified version of Blackjack. If you're not familiar with Blackjack, refer to this video for game rules. Our simplified rules are the following. Please read all the requirements before starting it!
+2. There will be only two players. One human and one computer (for the Base solution).
+3. The computer will always be the dealer.
+4. Each player gets dealt two cards to start.
+5. The player goes first, and decides if they want to hit (draw a card) or stand (end their turn).
+6. The dealer has to hit if their hand is below 17.
+7. Each players' score is the total of their card ranks. Jacks/Queen/Kings are 10. Aces can be 1 or 11.
+8. The player who is closer to, but not above 21 wins the hand.
+
+
+Gameplay Description
+=====================
+1. The main function runs on each player's turn. The sequence of actions in the game might be the following.
+2. Deck is shuffled.
+3. User clicks Submit to deal cards.
+4. The cards are analysed for game winning conditions, e.g. Blackjack.
+5. The cards are displayed to the user.
+6.The user decides whether to hit or stand, using the submit button to submit their choice.
+7. The user's cards are analysed for winning or losing conditions.
+8. The computer decides to hit or stand automatically based on game rules.
+9. The game either ends or continues.
+*/
+
+var playerDeck = {
```

Comment:
good job setting up your global states in preparation for the game. It helps to clarify what you're using for the game logic